modules/using-must-gather.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/using-the-must-gather-tool.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-running-must-gather_{context}"]
+= Using the must-gather tool
+
+You can run the `must-gather` tool with the following options. To use an option, you can add a flag corresponding to that option in the `must-gather` command.
+
+Default configuration:: This configuration collects pod logs, {oadp-short} and `Velero` custom resource (CR) information for all namespaces where the {oadp-short} Operator is installed.
+Timeout:: Data collection can take a long time if there are many failed `Backup` CRs. You can improve performance by setting a timeout value.
+Insecure TLS connections:: If a custom CA certificate is used, use the `must-gather` tool with  insecure TLS connections.
+
+The `must-gather` tool generates a Markdown output file with the collected information. The Markdown file is located in a cluster directory.
+
+.Prerequisites
+
+* You have logged in to the {product-title} cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+* You are using {oadp-short} 1.3 or 1.4.
+
+.Procedure
+
+. Navigate to the directory where you want to store the `must-gather` data.
+. Run the `oc adm must-gather` command for one of the following data collection options:
+
+* To use the default configuration of the `must-gather` tool, run the following command:
+
+** For {oadp-short} 1.3, run the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather-v1-3}
+----
++
+** For {oadp-short} 1.4, run the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather-v1-4}
+----
++
+* To use the timeout flag with the `must-gather` tool, run the following command:
+
+** For {oadp-short} 1.3, run the following command:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc adm must-gather --image={must-gather-v1-3} -- /usr/bin/gather --request-timeout 1m # <1>
+----
+<1> In this example, the timeout is 1 minute.
```

Comment:
Do we want to make it generic like the 1.4 command to be parallel?  What values are accepted: minute, seconds, etc?
```suggestion
[source,terminal,subs="attributes+"]
----
$ oc adm must-gather --image={must-gather-v1-3} -- /usr/bin/gather --request-timeout <timeout> # <1>
----
<1> Specify a timeout value.
```